### PR TITLE
Small fixes to layouts

### PR DESCRIPTION
**Pull request by andrewsilke**
- color picker Qt small color renames (it is not in use) fixes to account for bad rename in color refactor
- fixes to textModified for lineEdits to account for float comparisons.

### CODE REVIEW

File: zoo/libs/pyqt/widgets/layouts.py [hunk 1]
```diff
@@ -88,6 +88,15 @@ def __init__(self, parent=None):
         self.returnPressed.connect(self.__handleReturnPressed)
         self._before = ""
 
+    def __getBeforeAfter(self):
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: name mangling? only a single underscore if not intending the mangle

File: zoo/libs/pyqt/widgets/layouts.py [hunk 2]
```diff
@@ -88,6 +88,15 @@ def __init__(self, parent=None):
...
+        """returns the before state and the after
```
Comment (dsparrow27) [inline zoo/libs/pyqt/widgets/layouts.py]: capital 'R' :P

File: zoo/libs/pyqt/widgets/layouts.py (hunk 1 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: was a copy paste off the interweb, I have no idea why you'd use two underscores?

File: zoo/libs/pyqt/widgets/layouts.py (hunk 2 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: ha been really getting serious about naming I see, no worries, fixed will be in the next push. Might as well merge this now.

File: zoo/libs/pyqt/widgets/layouts.py (hunk 1 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: All good double underscores removed

File: zoo/libs/pyqt/widgets/layouts.py (hunk 2 above)
Comment (andrewsilke) [reply zoo/libs/pyqt/widgets/layouts.py]: done, this one will be in next push

**Pull request merged**